_includes/extensions/writing-repeater.html

```diff
@@ -27,158 +47,131 @@ <h2 id="writing-extensions-repeater">Repeater Views</h2>
+				from the dataSource, as well as a <code>callback</code> function. The <code>callback</code> function must
+				be called in order for the repeater to complete rendering. With this function, it is up to the view extension
+				developer to append contents to the <code>container</code> as they see fit.
 			</td>
 		</tr>
 		</tbody>
 	</table>
 </div><!-- ./fu-table-responsive -->
 
-<h3 id="writing-extensions-repeater-helpers">Helpers Object</h3>
-<p>The helpers object contains various attributes useful for rendering data:</p>
+<h3 id="writing-extensions-repeater-itemObject">Item Object</h3>
+<p>The <code>before</code>, <code>renderItem</code>, and <code>after</code> functions can be returned item objects, which
+	specify certain actions to be carried out by the repeater, such as inserting content:</p>
 <div class="table-responsive">
 	<table class="table table-bordered table-striped">
 		<thead>
 		<tr>
 			<th style="width: 100px;">Name</th>
-			<th style="width: 100px;">type</th>
+			<th style="width: 260px;">type</th>
 			<th>description</th>
 		</tr>
 		</thead>
 		<tbody>
 		<tr>
-			<td>container</td>
-			<td><a href="http://learn.jquery.com/using-jquery-core/jquery-object/">jQuery object</a></td>
-			<td>The parent container for the current renderer. Default value includes the <code>.repeater-canvas</code>
-				div. Remains true unless another container is specified via data-attributes.
-			</td>
-		</tr>
-		<tr>
-			<td>data</td>
-			<td>object</td>
-			<td>The data object as returned from the <code>dataSource</code> in its entirety</td>
-		</tr>
-		<tr>
-			<td>index</td>
-			<td>number</td>
-			<td>Only present if the <code>repeat</code> attribute value equals the current renderer. Specifies
-				the current index of the subset being iterated over.
-			</td>
+			<td>item</td>
+			<td>string, jQuery object, or DOM node</td>
+			<td>Specifies the content to be inserted into the <code>container</code>. If not specified, no action will be taken.</td>
 		</tr>
 		<tr>
-			<td>item</td>
-			<td>str or <a href="http://learn.jquery.com/using-jquery-core/jquery-object/">jQuery object</a></td>
-			<td>Represents the item returned from the <code>render</code> function of the current renderer. Only present
-				in functions following the <code>render</code> function when an <code>item</code> was returned.
+			<td>action</td>
+			<td>string</td>
+			<td>Optional. Specifies the jQuery mechanism of action to insert <code>item</code> content. If not specified, the default action will be
+				<code>'append'</code>. Any of the <a href="http://api.jquery.com/category/manipulation/dom-insertion-inside/">jQuery insertion methods</a>
+				can be used. If set to <code>'none'</code>, no insertion will occur.
 			</td>
 		</tr>
 		<tr>
-			<td>subset</td>
-			<td>array</td>
-			<td>Only present if the <code>repeat</code> attribute value equals the current renderer. Specifies the
-				subset iterated over.
+			<td>container</td>
```

Comment:
this should be the definitive definition for "container" which the others reference